staff/jordi-pujol/spa/index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
```

Comment:
You could use a title here, maybe something like "SPA".

Reply:
Thanks! I've just changed it 👍